test/integration/camera.cc

```diff
@@ -49,15 +49,25 @@
 #include "TransportTestTools.hh"
 
 std::mutex g_mutex;
-unsigned int g_imgCounter = 0;
+unsigned int g_imgCounter1 = 0;
+unsigned int g_imgCounter2 = 0;
 
-void OnGrayscaleImage(const gz::msgs::Image &_msg)
+void OnGrayscaleImageL8(const ignition::msgs::Image &_msg)
```

Comment:
Because of the transition to ``gz`` namespace, we should keep this as ``gz::msgs::Image``